### PR TITLE
Vbr/ktor descriptor dependencies

### PR DESCRIPTION
**Pull request by Ololoshechkin**

### CODE REVIEW

File: intellij-plugin-structure/structure-ktor/src/main/kotlin/com/jetbrains/plugin/structure/ktor/KtorFeature.kt [hunk 1]
```diff
@@ -16,8 +16,13 @@ data class KtorFeature(
   override val icons: List<PluginIcon> = emptyList(),
   override val pluginVersion: String? = null,
   override val pluginId: String? = null,
+  val ktorVersion: String? = null,
```
Comment (AlexanderPrendota) [inline intellij-plugin-structure/structure-ktor/src/main/kotlin/com/jetbrains/plugin/structure/ktor/KtorFeature.kt:19]: what the difference between ktorVersion and pluginVersion? probably it should be the same

File: intellij-plugin-structure/structure-ktor/src/main/kotlin/com/jetbrains/plugin/structure/ktor/KtorVersions.kt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.jetbrains.plugin.structure.ktor
+
+val KTOR_VERSIONS = listOf(
```
Comment (AlexanderPrendota) [inline intellij-plugin-structure/structure-ktor/src/main/kotlin/com/jetbrains/plugin/structure/ktor/KtorVersions.kt:3]: how you will update that set?

and maybe setOf()?

**Review by AlexanderPrendota — CHANGES_REQUESTED** [review]

File: intellij-plugin-structure/structure-ktor/src/main/kotlin/com/jetbrains/plugin/structure/ktor/KtorFeature.kt (hunk 1 above)
Comment (Ololoshechkin) [reply intellij-plugin-structure/structure-ktor/src/main/kotlin/com/jetbrains/plugin/structure/ktor/KtorFeature.kt:19]: No, `pluginVersion` is a version of the plugin (feature). And `ktorVersion` is a version of Ktor framework that this feature was compiled with.

File: intellij-plugin-structure/structure-ktor/src/main/kotlin/com/jetbrains/plugin/structure/ktor/KtorVersions.kt (hunk 2 above)
Comment (Ololoshechkin) [reply intellij-plugin-structure/structure-ktor/src/main/kotlin/com/jetbrains/plugin/structure/ktor/KtorVersions.kt:3]: `setOf()` -- Agree, will fix.

Regarding updates -- I think this should be updated each time a new version of Ktor releases (approx. each month). Does this seem possible?

Comment (AlexanderPrendota) [reply intellij-plugin-structure/structure-ktor/src/main/kotlin/com/jetbrains/plugin/structure/ktor/KtorVersions.kt]: too hard. I guess You will forget to update that field :(

Comment (Ololoshechkin) [reply intellij-plugin-structure/structure-ktor/src/main/kotlin/com/jetbrains/plugin/structure/ktor/KtorVersions.kt]: Do we have any other options?

Comment (AlexanderPrendota) [reply intellij-plugin-structure/structure-ktor/src/main/kotlin/com/jetbrains/plugin/structure/ktor/KtorVersions.kt]: validate ktor version on ktor-feature-generator?

**Review by AlexanderPrendota — APPROVED** [review]

**Pull request merged**